examples/wearable/UIComponents/contents/user-inputs-elements/button/index.html

```diff
@@ -26,40 +26,51 @@ <h2>
 		<div class="ui-content">
 			<ul class="ui-listview">
 				<li>
-					<a href="text-button.html">
-						Text
+					<a href="big-icon-button.html">
+						Big Icon
 					</a>
 				</li>
 				<li>
-					<a href="short-text-button.html">
-						Short Text
+					<a href="small-icon-button.html">
+						Small Icon
 					</a>
 				</li>
 				<li>
-					<a href="long-text-button.html">
-						Long Text
+					<a href="icon-button.html">
+						Icon
 					</a>
 				</li>
 				<li>
-					<a href="icon-button.html">
-						Icon
+					<a href="icon-button-from-js.html">
+						Icon From JS
 					</a>
 				</li>
 				<li>
-					<a href="small-icon-button.html">
-						Small Icon
+					<a href="bottom-button.html">
+						Bottom button
```

Comment:
Is it good to sync the naming rule?
Appendix all 'button' or remove the 'button'.

Reply:
I appendix all "button" string.
Thanks.